UPGRADE_5.0.md

```diff
@@ -294,3 +294,17 @@ The dropped methods are:
 * redirect
 * redirectToErrorPage
 * getConfig
+
+## Backend Header
+
+`Backend\Core\Engine\Header` also known as the service `header` when you are in the backend was stripped of some functions that where available because it extended a base class. 
+This is no longer the case.
+The dropped methods are:
+* getAction
```

Comment:
It is in the table below, but I've added a clarification for that below
50cb94814